Dagstuhl Research Online Publication Server.js

```diff
@@ -0,0 +1,166 @@
+		}
+		if (bibtexEntry.indexOf("@Article")>-1) {
+			return "journalArticle";
+		}
+		return "conferencePaper";
+	} else if (getSearchResults(doc, true)) {
+		return "multiple";
+	}
+}
+
+function getSearchResults(doc, checkOnly) {
+	var items = {};
+	var found = false;
+	var rows = ZU.xpath(doc, '//b/a[contains(@href, "source_opus")]|//td/a[contains(@href, "source_opus")]');//
+	for (var i=0; i<rows.length; i++) {
+		var href = rows[i].href;
+		var title = ZU.trimInternal(rows[i].textContent);
+		if (!href || !title) continue;
+		if (checkOnly) return true;
+		found = true;
+		items[href] = title;
+	}
+	return found ? items : false;
+}
+
+
+function doWeb(doc, url) {
+	if (detectWeb(doc, url) == "multiple") {
+		Zotero.selectItems(getSearchResults(doc, false), function (items) {
+			if (!items) {
+				return true;
+			}
+			var articles = [];
+			for (var i in items) {
+				articles.push(i);
+			}
+			ZU.processDocuments(articles, scrape);
+		});
+	} else {
+		scrape(doc, url);
+	}
+}
+
+function scrape(doc, url) {
+    var bibtexEntry = ZU.xpathText(doc, '//pre/tt');
+    //Z.debug(bibtexEntry);
+
+    var translator = Zotero.loadTranslator("import");
+    translator.setTranslator("9cb70025-a888-4a29-a210-93ec52da40d4");
+    translator.setString(bibtexEntry);
+    translator.setHandler("itemDone", function(obj, item) {
+        item.attachments.push({
+            title: "Snapshot",
+            document: doc
+        });
+        item.complete();
+    });
+    translator.translate();         
+
+}/** BEGIN TEST CASES **/
+var testCases = [
+	{
+		"type": "web",
+		"url": "http://drops.dagstuhl.de/opus/frontdoor.php?source_opus=4958",
+		"items": [
+			{
+				"itemType": "conferencePaper",
+				"title": "Overcoming Intractability in Unsupervised Learning (Invited Talk)",
+				"creators": [
+					{
+						"firstName": "Sanjeev",
+						"lastName": "Arora",
+						"creatorType": "author"
+					},
+					{
+						"firstName": "Ernst W.",
+						"lastName": "Mayr",
+						"creatorType": "editor"
+					},
+					{
+						"firstName": "Nicolas",
+						"lastName": "Ollinger",
+						"creatorType": "editor"
+					}
+				],
+				"date": "2015",
+				"DOI": "http://dx.doi.org/10.4230/LIPIcs.STACS.2015.1",
+				"ISBN": "9783939897781",
+				"itemID": "arora:LIPIcs:2015:4958",
+				"libraryCatalog": "Dagstuhl Research Online Publication Server",
+				"pages": "1–1",
+				"place": "Dagstuhl, Germany",
+				"proceedingsTitle": "32nd International Symposium on Theoretical Aspects of Computer Science (STACS 2015)",
+				"publisher": "Schloss Dagstuhl–Leibniz-Zentrum fuer Informatik",
+				"series": "Leibniz International Proceedings in Informatics (LIPIcs)",
+				"url": "http://drops.dagstuhl.de/opus/volltexte/2015/4958",
+				"volume": "30",
+				"attachments": [
+					{
+						"title": "Snapshot"
+					}
+				],
+				"tags": [],
+				"notes": [
+					{
+						"note": "<p>Keywords: machine learning, unsupervised learning, intractability, NP-hardness</p>"
```

Comment:
I checked a couple of notes and they all appear to contain keywords in a standardized format (i.e. Keyword: followed by a comma separated list). Let's test for that format (i.e. for the "Keywords: " part, and if it matches, strip Keywords:\s*, add the keywords as tags and delete the note.